stats/cloud/api.go

```diff
@@ -112,3 +116,30 @@ func (c *Client) TestFinished(referenceID string, thresholds ThresholdResult, ta
 	err = c.Do(req, nil)
 	return err
 }
+
+func (c *Client) Login(email string, password string) (*LoginResponse, error) {
+	url := fmt.Sprintf("%s/login", c.baseURL)
+
+	data := struct {
+		Email    string `json:"email"`
+		Password string `json:"password"`
+	}{
+		email,
+		password,
+	}
+
+	req, err := c.NewRequest("POST", url, data)
+	if err != nil {
+		return nil, err
+	}
+
+	lr := LoginResponse{}
+	err = c.Do(req, &lr)
+	if err != nil {
+		// if wrong email or password, it will throw an Error
+		// `client.Do` should handle this in a better way
```

Comment:
How so...? An error seems appropriate.

Also: capital letters at the start of sentences, periods at the end.